ci_verify.sh

```
@@ -70,12 +156,14 @@ do
 
         # This will fail and exit if the ioc.yaml is invalid
         # Also show the startup script we just generated (and verify it exists)
+        # 'ibek runtime generate2 /config' reads the whole mounted config folder
+        # (ioc.yaml + any vendored/local *.ibek.support.yaml, proto and db) and
+        # places the generated runtime (st.cmd, proto, db) under /epics/runtime.
         $docker run --rm --entrypoint bash \
             -v ${service}/config:/config:z \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)